src/wfs/src/main/java/org/geoserver/wfs/xml/FeatureTypeSchemaBuilder.java

```diff
@@ -640,18 +640,12 @@ boolean findTypeInSchema(FeatureTypeInfo featureTypeMeta, XSDSchema schema, XSDF
                     Schemas.findSchemaLocationResolvers(xmlConfiguration);
             List<XSDSchemaLocator> locators = new ArrayList<>();
             locators.add(
-                    new XSDSchemaLocator() {
-                        public XSDSchema locateSchema(
-                                XSDSchema schema,
-                                String namespaceURI,
-                                String rawSchemaLocationURI,
-                                String resolvedSchemaLocationURI) {
-
-                            if (gmlNamespace.equals(namespaceURI)) {
-                                return gmlSchema();
-                            }
-                            return null;
+                    (schema1, namespaceURI, rawSchemaLocationURI, resolvedSchemaLocationURI) -> {
+
+                        if (gmlNamespace.equals(namespaceURI)) {
+                            return gmlSchema();
                         }
+                        return null;
```

Comment:
Probably a little easier to understand the old way.

Reply:
Agreed